_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -84,7 +85,7 @@ There are no accessibility support issues known.
 
 #### Passed Example 1
 
-These two HTML `a` elements have the same [accessible name][] and [context][programmatically determined link context] and link to the [same resource][].
+These two HTML `a` elements have the same non-empty [accessible name][], empty [accessible description][], the same [context][programmatically determined link context], and resolve to the [same resource][].
```

Comment:
I think these texts are gettin pretty verbose. Not a blocker if people prefer this language, but I would suggest this:

```suggestion
These two links with the same [name][]accessible name], [description][]accessible description] and [context][programmatically determined link context] resolve to the [same resource][].
```

The same for other example descriptions.